docs/book/src/quick-start.md

```
@@ -56,7 +56,7 @@ This process may take some time and, once the provider is running, you'll be abl
 
 #### Cluster API Operator
 
-You can refer to the Cluster API Operator book [here](https://cluster-api-operator.sigs.k8s.io/01_user/02_quick-start) to learn about the basics of the project and how to install the operator.
+You can refer to the Cluster API Operator book [here](https://cluster-api-operator.sigs.k8s.io/user/quick-start) to learn about the basics of the project and how to install the operator.
```

markdownlint-cli2 (0.23.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)